data/coalition/coalition outfits.txt

```diff
@@ -224,6 +224,17 @@ outfit "Scanning Module"
 	"tactical scan power" 25
 	description "This system allows the Heliarchs to scan nearby ships, enabling them to detect weapons or illegal goods."
 
+outfit "Outskirts Gauger"
+	category "Systems"
+	cost 1406000
+	thumbnail "outfit/outskirts gauger"
+	"mass" 8
+	"outfit space" -8
+	"tactical scan power" 256
+	"energy consumption" .25
+	"energy capacity" -200
```

Comment:
```suggestion
	"mass" 4
	"outfit space" -4
	"asteroid scan power" 40
	"outfit scan power" 36
	"outfit scan speed" 1
	"cargo scan power" 36
	"cargo scan speed" 1
	"tactical scan power" 30
	"energy consumption" .12
	"energy capacity" -100
```
Reasons:
- A very solid upgrade over their existing scanner module,
- Gives them new capabilities that fit the "running calculations about every little thing within its impressive range" description.
- Retains the modular nature.
- More general purpose and less specialized on a facet that is useless to the Coalition.